proto/exocore/assets/v1/tx.proto

```diff
@@ -124,24 +124,16 @@
     (gogoproto.nullable) = false
   ];
 
-  // operator_amount is the amount that the operator owns.
-  //todo: the field is used to mark operator's own assets and is not temporarily used now
-  string operator_amount = 2
-  [
-    (cosmos_proto.scalar) = "cosmos.Int",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable) = false
-  ];
   // wait_unbonding_amount is the amount that is waiting for unbonding.
-  string wait_unbonding_amount = 3
+  string wait_unbonding_amount = 2
   [
     (cosmos_proto.scalar) = "cosmos.Int",
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable) = false
   ];
 
   // operator_unbonding_amount is the amount that is owned by operator itself and waiting for unbonding.
-  string operator_unbonding_amount = 4
+  string operator_unbonding_amount = 3
   [
     (cosmos_proto.scalar) = "cosmos.Int",
```

Comment:
I didn't rename `operator_unbonding_amount` to `wait_unbonding_amount`, because the above useless field has been removed. Additionally, the field `operator_unbonding_amount` is useless, so I will remove it in the subsequent commit.